### PR TITLE
Fix#1647: Bug in Albums mode resolved

### LINKED ISSUE #1647: Bug in onLongpressed in albums mode
**Actual Behaviour**
Currently in the photos mode when we select a picture by long pressing it and select another picture by long pressing, then all the photos lying in between those indices are selected. But when the same thing is applied to albums mode or show all photos, it doesn't select the intermediate photos.

**Expected Behaviour**
It should select the photos lying between the two indices.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixed #1647 

Changes:Earlier when an album was selected by long pressing it and then another album was also selected by long pressing it , the range of albums in between them wasn't selected . I have made sure that this feature which was earlier available for the photos mode is also available for albums mode as well.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/data/HandlingAlbums.java [hunk 1]
```diff
@@ -312,6 +317,30 @@ public void sortAlbums(final Context context) {
...
+        if (dispAlbums.get(index) != null && !dispAlbums.get(index).isSelected()) {
+            dispAlbums.get(index).setSelected(true);
+            selectedAlbums.add(dispAlbums.get(index));
+            adapter.notifyItemChanged(index);
+        }
+      }
+    }
+  }
+
+
```
Comment (mayank8318) [inline app/src/main/java/org/fossasia/phimpme/gallery/data/HandlingAlbums.java]: Please avoid extra spaces :)

**Comment by sridharjajoo** [comment]
@mayank8318 Done the changes!

**Comment by codedsun** [comment]
@sridharjajoo Please resolve the conflicts.

**Comment by sridharjajoo** [comment]
@mayank8318 @abishekvashok Kindly review

**Comment by sridharjajoo** [comment]
@abishekvashok Kindly merge this!

File: app/src/main/java/org/fossasia/phimpme/gallery/data/HandlingAlbums.java [hunk 2]
```diff
@@ -312,6 +317,27 @@ public void sortAlbums(final Context context) {
...
   }
 
+  public void selectAllPhotosUpToAlbums(int targetIndex, AlbumsAdapter adapter) {
+    int indexRightBeforeOrAfter = -1;
+    int indexNow;
+    for (Album selectedAlbum : selectedAlbums) {
+      indexNow = dispAlbums.indexOf(selectedAlbum);
+      if (indexRightBeforeOrAfter == -1) indexRightBeforeOrAfter = indexNow;
+
+      if (indexNow > targetIndex) break;
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/gallery/data/HandlingAlbums.java]: Properly intend this line to another line pls.

File: app/src/main/java/org/fossasia/phimpme/gallery/data/HandlingAlbums.java [hunk 3]
```diff
@@ -312,6 +317,27 @@ public void sortAlbums(final Context context) {
     }
   }
 
+  public void selectAllPhotosUpToAlbums(int targetIndex, AlbumsAdapter adapter) {
+    int indexRightBeforeOrAfter = -1;
+    int indexNow;
+    for (Album selectedAlbum : selectedAlbums) {
+      indexNow = dispAlbums.indexOf(selectedAlbum);
+      if (indexRightBeforeOrAfter == -1) indexRightBeforeOrAfter = indexNow;
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/gallery/data/HandlingAlbums.java]: Please move the if body to a new line

File: app/src/main/java/org/fossasia/phimpme/gallery/data/HandlingAlbums.java (hunk 2 above)
Comment (sridharjajoo) [reply app/src/main/java/org/fossasia/phimpme/gallery/data/HandlingAlbums.java]: Done the changes

**Comment by sridharjajoo** [comment]
@isuruAb @abishekvashok Kindly merge this P.R..Thanks!

**Comment by codedsun** [comment]
@abhishekvashok merge this! @sridharjoo update the branch

**Comment by sridharjajoo** [comment]
@anantprsd5 Kindly merge this!

**Comment by sridharjajoo** [comment]
@abishekvashok Kindly merge this

**Comment by isuruAb** [comment]
@abishekvashok, please review this PR

**Pull request merged**